cmd/deploy/deploy.go

```diff
@@ -73,8 +75,13 @@ func main() {
 	if !common.IsHexAddress(*ownerAddressString) {
 		panic("please specify a valid rollup owner address")
 	}
+	if !common.IsHexAddress(*loserEscrowAddressString) {
+		panic("please specify a valid loser escrow address")
```

Comment:
Maybe we can allow the default if !prod

Reply:
We should also probably require that the chain id isn't the default if prod is set

Reply:
I think the default for this is fine since an empty string is valid for `IsHexAddress` iirc

Reply:
It checks the length:
```go
	return len(s) == 2*AddressLength && isHex(s)
```
I think you're thinking of `common.HexToAddress` which does accept an empty string, meaning if we simply let an empty string pass this check if `!prod` it should work.

Reply:
Done